runtime/lua/vim/lsp/handlers.lua

```diff
@@ -468,15 +485,8 @@ M['window/logMessage'] = function(_, result, ctx, _)
   local client_name = client and client.name or string.format('id=%d', client_id)
   if not client then
     err_message('LSP[', client_name, '] client has shut down after sending ', message)
-  end
-  if message_type == protocol.MessageType.Error then
-    log.error(message)
-  elseif message_type == protocol.MessageType.Warning then
-    log.warn(message)
-  elseif message_type == protocol.MessageType.Info or message_type == protocol.MessageType.Log then
-    log.info(message)
```

Comment:
how frequent is `window/logMessage` in a typical use sessin? the reason we only display ERROR logs is because most users won't want/need to see non-error messages.

Reply:
If users don't want to see logs lower than ERROR, than they should set it with `vim.lsp.log.set_level()`

Reply:
No, that affects what gets logged to the logfile. That is different than saying "I don't want all logs displayed in the UI".